src/phoenix/server/api/types/Experiment.py

```diff
@@ -21,12 +21,26 @@
 
 @strawberry.type
 class Experiment(Node):
+    ordinality: Private[Optional[int]] = None
```

Comment:
```suggestion
    cached_sequence_number: Private[Optional[int]] = None
```

Suggestion: One fewer concept.

Reply:
updated